python/rmm/CMakeLists.txt

```diff
@@ -30,4 +30,5 @@ rapids_cython_init()
 add_compile_definitions("SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${RMM_LOGGING_LEVEL}")
 
 add_subdirectory(rmm/_cuda)
-add_subdirectory(rmm/_lib)
+add_subdirectory(rmm/librmm)
```

Comment:
OK, so this is the cython "headers".

*question*: If we want `librmm` "headers" to be available to other packages, do we need some pyproject.toml changes to ensure they're installed?

*meta-question*: Is that something we are trying to achieve/want?

Reply:
Yes, we do need to make those available. RAPIDS has set the precedent for libraries using rmm Python objects as well as librmm C++ objects and needing to extract the latter from the former to pass down via Cython. Moreover, as we add support for streams/mrs in Python that usage is only going to become more common. We're going to have to lean into that.

We don't need to do anything special for this, though. SKBC will automatically include all files that are under source control unless explicitly told not to. If I download one of the wheels produced in this PR and look, I can see the necessary files:
```
(main) dt08% unzip -l rmm_cu12-24.12.0a24-cp310-cp310-manylinux_2_24_x86_64.manylinux_2_28_x86_64.whl | grep "/librmm/.*pxd"
     1170  2024-10-01 19:15   rmm/librmm/cuda_stream_view.pxd
     2115  2024-10-01 19:15   rmm/librmm/device_buffer.pxd
     1658  2024-10-01 19:15   rmm/librmm/device_uvector.pxd
     1027  2024-10-01 19:15   rmm/librmm/cuda_stream.pxd
     2232  2024-10-01 19:15   rmm/librmm/_logger.pxd
     1349  2024-10-01 19:15   rmm/librmm/per_device_resource.pxd
     8780  2024-10-01 19:15   rmm/librmm/memory_resource.pxd
      932  2024-10-01 19:15   rmm/librmm/cuda_stream_pool.pxd
```

Reply:
What is SKBC?

Reply:
Yes sorry I've gotten used to the abbreviation in some places and started using it everywhere.